mne/minimum_norm/time_frequency.py

```diff
@@ -406,22 +552,33 @@ def _source_induced_power(
             with_power=True,
             pick_ori=pick_ori,
             decim=decim,
+            noise_norm=noise_norm,
         )
         for data in np.array_split(epochs_data, n_jobs)
     )
-    power = sum(o[0] for o in out)
+    power = sum(o[0] for o in out)  # power shape: (n_verts, n_freqs, n_samps)
     power /= len(epochs_data)  # average power over epochs
 
+    if isinstance(label, (Label, BiHemiLabel)):
+        logger.info(
+            f"Outputting power for {len(power)} vertices in label" f" {label.name}."
+        )
+    elif isinstance(label, list):
+        power = _get_label_power(power, label, vertno, k_id)
+        logger.info(
+            "Averaging induced power across vertices within labels "
+            f"for {len(label)} labels."
```

Comment:
this will be "for 1 label" or "for 2 labels" for example
```suggestion
            f"for {len(label)} label{_pl(label)}."
```